### ISSUE TITLE
[BUG]: Log file location

### ISSUE THREAD

**Issue by fadnincx** [projectpotos/ansible-plays-potos#13]
### Description of the bug

Currently the log is always written to `/var/log/potos/ansible.log` On a customer client this directory doesn't exists and we probably want to write the log to `/var/log/<clientname>/ansible.log`

### Steps To Reproduce

-

**Comment by nis65** [comment]
I am hit with this bug in my client too. My current workaround is to
do after installation on the client:

```
ln -s /var/log/mlc /var/log/potos
```
Then I get the logs back from `mlc-ansible-pull`:

```
root@mlc:/var/log# ls -l mlc
-rw-r--r-- 1 root root 195054 Apr 30 09:00 ansible.log
-rw-r--r-- 1 root root      0 Apr 30 00:00 setup.log
-rw-r--r-- 1 root root   3810 Apr 22 16:45 setup.log.1.gz
```

Would be nice to have that fixed, but I am unsure what exactly the desired situation is...

**Comment by fadnincx** [comment]
Yes that would be a workaround. Tbh I haven't thought this issue through 100% yet. however, I would find a solution without there being `/var/log/potos` the nicest. This would however require `ansible.cfg` to be (at least partially) configurable (probably a good idea too :thinking:)

**Comment by fadnincx** [comment]
At least for daily execution should be fixed with [ansible-role-potos_basics#14](https://github.com/projectpotos/ansible-role-potos_basics/pull/14)
